orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/DeregisterInstancesFromLoadBalancerTask.groovy

```diff
@@ -17,12 +17,39 @@
 
 package com.netflix.spinnaker.orca.clouddriver.tasks.instance
 
+import com.netflix.spinnaker.orca.TaskResult
+import com.netflix.spinnaker.orca.clouddriver.pipeline.servergroup.support.Location
+import com.netflix.spinnaker.orca.clouddriver.utils.MonikerHelper
+import com.netflix.spinnaker.orca.clouddriver.utils.TrafficGuard
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Component
 
 @Component
 class DeregisterInstancesFromLoadBalancerTask extends AbstractInstanceLoadBalancerRegistrationTask {
+  @Autowired
+  TrafficGuard trafficGuard
```

Comment:
I'd suggest doing an `@Autowired` constructor instead of this ... trying to be consistent around that moving forward.

I realize everywhere else you looked likely had the old style 🍤